accepted/0000-isolated-workspaces.md

```diff
@@ -46,37 +47,30 @@ If you run a command with the isolated flag and are in a workspaceDir, it will r
 
 If you run a command with the isolated flag but you are at the rootDir (or any other directory not underneath a workspaceDir), it will give an error saying that the isolated flag can only be used inside an individual workspace.
 
-## Command behavior
+## Command behavior with isolated flag
 ### install
-#### With isolated flag
 Install only that package's dependencies, and do not hoist them to the rootDir. Install dependencies on other
 workspaces from registry instead of using the local version. 
 
 It should still use the rootDir's lockfile since adding lockfiles for each workspace will cause conflicts
 across workspaces. Since other workspaces will not be in the lockfile, it should determine their versions as if it was 
 installing without a lockfile (by taking the newest version that matches their semver).
 
-#### Changes to current (non-isolated) behavior
```

Comment:
deleting the dependency on install seems to happen automatically already, so it was unnecessary. With that, no existing commands needed changing.